src/Datasource/Exception/PageOutOfBoundsException.php

```diff
@@ -22,12 +22,12 @@ class PageOutOfBoundsException extends Exception
     /**
      * {@inheritDoc}
      */
-    protected $_messageTemplate = 'Page number "%s" could not be found.';
+    protected $_messageTemplate = 'Page number %s could not be found.';
 
     /**
      * Constructor
      *
-     * @param string|null $message The error message.
+     * @param array|null $message Paging info.
```

Comment:
Is it allowed for some exceptions to have an array here instead of string as argument?
Couldnt this break some handlers that do not expect this?